angular-workspace/projects/example-client-app/src/app/app.module.ts

```diff
@@ -62,7 +64,8 @@ import { HeaderComponent } from './header/header.component';
         RouterModule.forRoot(
```

Comment:
Could you set `title` for these pages too so that the page title changes as you navigate? https://angular.io/guide/router#setting-the-page-title

Reply:
Done